src/esp/gfx/CubeMapCamera.cpp

```diff
@@ -49,38 +49,47 @@ CubeMapCamera& CubeMapCamera::switchToFace(unsigned int cubeSideIndex) {
   return *this;
 }
 
-CubeMapCamera& CubeMapCamera::switchToFace(Mn::GL::CubeMapCoordinate cubeSide) {
+Mn::Matrix4 CubeMapCamera::getCameraLocalTransform(
```

Comment:
Create a new function to replace the internal lambda since it will be used in the future in the PBR IBL.